examples/wdio.conf.js

```diff
@@ -26,6 +26,10 @@ exports.config = {
     user: 'webdriverio',
     key:  'xxxxxxxxxxxxxxxx-xxxxxx-xxxxx-xxxxxxxxx',
     //
+    // If you run your tests on SauceLabs you can specify the region you want to run your tests
+    // in via the `region` property.
+    region: 'eu', // for eu-central-1
```

Comment:
For now there's only one region, but this is not clear in the docs. By providing a string I would, as a user, assume I can also use `us/usa` and so on. What's the chance that there will be a new region before a new major release? 
Wouldn't a `boolean` be better here?

Reply:
> Wouldn't a `boolean` be better here?

I would say no because I would have to change the code again. The reason I introduced short regions is because some people aren't familiar with region names.

Reply:
I can also just remove the short version and have people to put in the full region name

Reply:
I would suggest to just add this in the WDIO-conf / documentation

```js
    // If you run your tests on SauceLabs you can specify the region you want to run your tests
    // in via the `region` property.
    // Regions: eu|us|..
    region: 'eu',
```

Like you have in the `logLevel`-docs

```js
    // Level of logging verbosity: trace | debug | info | warn | error
    logLevel: 'info',
```